x-pack/plugins/reporting/server/lib/esqueue/constants/index.js

```diff
@@ -4,8 +4,8 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { statuses } from '../../statuses';
```

Comment:
Another small nit, if we care about alphabetizing, import order is wrong

Reply:
I'm using `shift+alt+o` in VS Code to automate sorting imports. It works by organizing on a few different dimensions, like relative distance from the current file, placing multi-imports before single-imports, etc.

However - the next two lines are out of order!

Reply:
TIL!